src/adapters/alpheiostb/adapter.js

```diff
@@ -24,31 +24,31 @@ class AlpheiosTreebankAdapter extends BaseAdapter {
    *      - {undefined} - if failed
   */
   async getHomonym (languageID, wordref) {
-    let server = this.prepareRequest(wordref)
+    const server = this.prepareRequest(wordref)
     if (!server.url) {
-      this.addError(this.l10n.messages['MORPH_TREEBANK_NO_URL'].get(wordref))
+      this.addError(this.l10n.messages.MORPH_TREEBANK_NO_URL.get(wordref))
       return
     }
     try {
-      let res = await this.fetch(server.url, { type: 'xml' })
+      const res = await this.fetch(server.url, { type: 'xml' })
 
       if (res.constructor.name === 'AdapterError') {
         return
       }
 
       if (res) {
-        let langCode = LMF.getLanguageCodeFromId(languageID)
+        const langCode = LMF.getLanguageCodeFromId(languageID)
 
-        let jsonObj = xmlToJSON.parseString(res)
+        const jsonObj = xmlToJSON.parseString(res)
         jsonObj.words[0].word[0].entry[0].dict[0].hdwd[0]._attr = { lang: { _value: langCode } }
 
-        let homonym = this.transform(jsonObj, jsonObj.words[0].word[0].form[0]._text, server.config)
+        const homonym = this.transform(jsonObj, jsonObj.words[0].word[0].form[0]._text, server.config)
         return homonym
       } else {
-        this.addError(this.l10n.messages['MORPH_TREEBANK_NO_ANSWER_FOR_WORD'].get(wordref))
+        this.addError(this.l10n.messages.MORPH_TREEBANK_NO_ANSWER_FOR_WORD.get(wordref))
```

Comment:
I did not even realize there was a change there: I was not touching this file and this line was fixed automatically by ESLint. This is a "Dot Notation" rule: https://eslint.org/docs/rules/dot-notation. I do not have a strong opinion on this, but I probably tend to agree with the arguments of the rule's creators. But I'm flexible here and can follow whatever we'll decide to use. @balmas, what's your opinion on this?

Reply:
I don't feel too strongly either. I see @irina060981 's point and also since there are circumstances when you can't use dot notation (such as when a variable is referenced or a reserved word) I think we wouldn't be able to switch entirely to dot notation over brackets.  So why don't we disable the rule for now?

Reply:
Added an exception for the "dot-notation" rule.